ballerina/modules/lexer/tests/test_utils.bal

```diff
@@ -16,7 +16,7 @@ import ballerina/test;
 
 # Returns a new lexer with the configured line for testing
```

Comment:
Not relevant to this PR, but why hv we added doc-comments to the test cases ?

Reply:
Yes, these doc comments are unnecessary. Will remove it in a separate PR.